### ISSUE TITLE
Error while trying to use multiplatform function on iOS.

### ISSUE THREAD

**Issue by agatap2** [GitLiveApp/firebase-kotlin-sdk#41]
I have written login and register functions in kotlin multiplatform project. Then I imported this project to android and ios applications as separate module. On android it works perfectly fine but I get this error while trying to use functions on iOS:

`Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'The default FIRApp instance must be configured before the default FIRAuthinstance can be initialized. One way to ensure that is to call [FIRApp configure]; (FirebaseApp.configure() in Swift) in the App Delegate's application:didFinishLaunchingWithOptions: (application(_:didFinishLaunchingWithOptions:) in Swift).`

I have added FirebaseApp.configure() to App Delegate but it didn't helped.

**Comment by nbransby** [comment]
Have you followed the instructions for setting iOS app in firebase, including add the plist?

**Comment by agatap2** [comment]
> Have you followed the instructions for setting iOS app in firebase, including add the plist?

Yes.

**Comment by nbransby** [comment]
Can you upload a reproducer ?

**Comment by agatap2** [comment]
I was following this tutorial while creating my multiplatform project: https://medium.com/better-programming/a-multi-platform-library-with-kotlin-episode-1-6c0f46d49944
And next I created firebase project for android and ios apps following google instructions.

Maybe problem is in fact that in AppDelegate.swift file I didn't imported Firebase library but FirebaseCore library. For some reason this are the only firebase libraries I can import: FirebaseCore, FirebaseCoreDiagnostics, FirebaseAuth, FirebaseAnalytics. Firebase import gives me "No such module 'Firebase'".

My Podfile:
```
target 'CurtainsIOS' do
  use_frameworks!

  pod 'Firebase'
  pod 'Firebase/Core'
  pod 'Firebase/Analytics'
  pod 'Firebase/Auth'

  target 'CurtainsIOSTests' do
    inherit! :search_paths
  end
  target 'CurtainsIOSUITests' do
  end
end
```

This is my first time programming iOS application.

**Comment by Elkhoudiry** [comment]
try to remove  this might help

  pod 'Firebase'
  pod 'Firebase/Core'

Firebase/core now comes bundled in with your libraries

**Comment by agatap2** [comment]
Thanks, Firebase module is available now but I still receive NSInternalInconsistencyException exception while trying to login with e-mail and password.

**Comment by Elkhoudiry** [comment]
since Firebase module available now, you can use FirebaseApp.configure() before you sign in,
and make sure your firebase file .plist is configured correctly with your project.
hope this will help

**Comment by nbransby** [comment]
THis should be fixed in 1.0.0
